assets/helpers/internationalisation/contributions.js

```diff
@@ -21,86 +19,30 @@ const countryGroupSpecificDetails: {
   GBPCountries: {
     headerCopy: defaultHeaderCopy,
     contributeCopy: defaultContributeCopy,
-    currency: {
-      name: 'British Pounds',
-      symbol: '£',
-    },
-    contribution: {
-      oneoff: 'One-off',
-      monthly: 'Monthly',
-    },
   },
   EURCountries: {
     headerCopy: defaultHeaderCopy,
     contributeCopy: defaultContributeCopy,
-    currency: {
-      name: 'Euros',
-      symbol: '€',
-    },
-    contribution: {
-      oneoff: 'One-off',
-      monthly: 'Monthly',
-    },
   },
   UnitedStates: {
     headerCopy: defaultHeaderCopy,
     contributeCopy: defaultContributeCopy,
-    currency: {
-      name: 'US Dollars',
-      symbol: '$',
-    },
-    contribution: {
-      oneoff: 'One-time',
-      monthly: 'Monthly',
-    },
   },
   AUDCountries: {
     headerCopy: 'Help us deliver the independent journalism Australia needs',
     contributeCopy: defaultContributeCopy,
-    currency: {
-      name: 'Australian Dollars',
-      symbol: '$',
-    },
-    contribution: {
-      oneoff: 'One-off',
-      monthly: 'Monthly',
-    },
   },
   International: {
     headerCopy: defaultHeaderCopy,
     contributeCopy: defaultContributeCopy,
-    currency: {
-      name: 'Canadian Dollars',
-      symbol: '$',
-    },
-    contribution: {
-      oneoff: 'One-off',
-      monthly: 'Monthly',
-    },
   },
   NZDCountries: {
     headerCopy: defaultHeaderCopy,
     contributeCopy: defaultContributeCopy,
-    currency: {
-      name: 'Canadian Dollars',
-      symbol: '$',
-    },
-    contribution: {
-      oneoff: 'One-off',
-      monthly: 'Monthly',
-    },
   },
   Canada: {
     headerCopy: defaultHeaderCopy,
     contributeCopy: defaultContributeCopy,
-    currency: {
-      name: 'Canadian Dollars',
-      symbol: '$',
-    },
-    contribution: {
-      oneoff: 'One-time',
-      monthly: 'Monthly',
-    },
   },
 };
 
```

Comment:
Would it make sense for this to live in `helpers/contributions`? Or would it make sense for some of the internationalisation logic there to live here? @joelochlann @Mullefa @tsop14 @jranks123